packages/salesforcedx-vscode-core/test/vscode-integration/util/inputUtils.test.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { expect } from 'chai';
+import { createSandbox, SinonSandbox } from 'sinon';
+import * as vscode from 'vscode';
+import { getTrimmedString } from '../../../src/util/inputUtils';
+
+describe('getTrimmedString', () => {
```

Comment:
nit: Can we have this be called `inputUtils Unit tests`?

Reply:
could probably just add a top level describe with that name and keep it as is.  Will help when we add a new function to inputUtils and want to write tests for it.